src/redux/store.js

```diff
@@ -0,0 +1,12 @@
+import { configureStore } from '@reduxjs/toolkit';
+import bookReducer from './books/bookSlice';
+import categoriesReducer from './categories/categoriesSlice';
+
+const store = configureStore({
+  reducers: {
+    books: bookReducer,
+    categories: categoriesReducer,
+  },
+});
```

Comment:
- [ ] KIndly make sure the `configureStore` have the parameter `reducer` which is a single function, it will be directly used as the root reducer for the store. Kindly make sure instead of naming it `reducers`  change its name to `reducer`. Kindly check this[ lesson](https://redux-toolkit.js.org/api/configureStore) for more info.